neqo-transport/src/connection.rs

```diff
@@ -2568,6 +2582,46 @@ mod tests {
         assert!(server.crypto.tls.info().unwrap().resumed());
     }
 
+    fn connect_with_rtt(client: &mut Connection, server: &mut Connection, now: &mut Instant) {
+        const TIME_INTERVAL: u64 = 20;
```

Comment:
I think that you should make this accessible on the outer scope and make it type `Duration`.  Maybe call it "RTT" and advance by half that value in this function.

Reply:
Or, you could pass the RTT as an argument.

There is a reason you might want to do that below.